draft-ietf-v6ops-464xlaton.xml

```diff
@@ -316,10 +353,8 @@ An ICMPv6 error packet from a global IPv6 address (not belonging to the NAT64 pr
 </li>
 </ul>
 <t>
-Using a dedicated IPv6 source address for CLAT traffic allows the node to make that distinction without keeping state, so CLAT can operate in the stateless mode (see Section 1.3 of <xref target="RFC7915"/>.
-In the absence of a dedicated address the CLAT instance would be required to use stateful NAT64.
-At the same time [RFC6877] defines 464XLAT as an architecure where the customer-side translation is stateless (it is even mentioned explictly in the [RFC6877] title.
-While it is possible to implement stateful client-side trabslation and use a single IPv6 address for both native and translated IPv6 packets, that approach doesn't fit the framework defined in [RFC6877] and, therefore, is out of scope of this document.
+Therefore, in the absence of dedicated addresses, the architecture would need additional stateful elements on the client side which are not part of 464XLAT as defined in <xref target="RFC6877"/>.
```

Comment:
Non-blocking comment: is it worth tying this to the existing NAT44 mentions in the doc? I think we should either not mention NAT44 at all which would be in line with what this line says (changes needed elsewhere in PR) or just reference that here with this:

```suggestion
Therefore, in the absence of dedicated addresses, the architecture would need additional stateful elements on the client side, such as NAT44, which are not part of 464XLAT as defined in <xref target="RFC6877"/>.
```

Reply:
Wait, no. NAT44 doesn't solve this issue - or maybe I misunderstand your comment?